src/htmlminifier.js

```diff
@@ -481,18 +481,19 @@
   }
 
   function canRemoveElement(tag, attrs) {
-    if (tag === 'textarea') {
-      return false;
-    }
-
-    if (tag === 'script') {
-      for (var i = attrs.length - 1; i >= 0; i--) {
-        if (attrs[i].name === 'src') {
-          return false;
+    switch (tag) {
+      case 'audio':
+      case 'iframe':
+      case 'textarea':
+      case 'video':
+        return false;
+      case 'script':
```

Comment:
I was being conservative - but yeah let's go with yours.

Can you think of other tags while we are at this?